bazelrio/libraries/wpilibc/BUILD

```diff
@@ -7,6 +7,7 @@ cc_library(
         "//libraries/ntcore",
         "//libraries/wpimath",
         "//libraries/wpiutil",
+        "//libraries/cscore",
```

Comment:
Not explicitly needed, so I would leave it to the user to add it. Maybe the "turn this into a bazel project" could add it by default. but I don't think it should be explicitly required. I have a completely uneducated hunch that if you are using C++, you aren't using opencv on the roborio (with the exception below).
 
They do some [witchcraft](https://github.com/wpilibsuite/allwpilib/blob/main/wpilibc/src/main/native/cppcs/RobotBase.cpp#L147) around the camera server, so probably the same thing there (which also is missing). This seems more likely to be used if you just want to pipe a usb webcam to the dashboard.

Reply:
The downside of including it is performance? Isn't it negligent?